### PR TITLE
python3Packages.django-bootstrap5: init at 23.3

### PR DESCRIPTION
**Pull request by NetaliDev**
## Description of changes
Adds the `django-bootstrap5` python module.

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 247896` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.django-bootstrap5</li>
    <li>python310Packages.django-bootstrap5.dist</li>
    <li>python311Packages.django-bootstrap5</li>
    <li>python311Packages.django-bootstrap5.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/django-bootstrap5/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+, beautifulsoup4
+, pillow
+, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-bootstrap5";
+  version = "23.3";
+  format = "pyproject";
+
```
Comment (Janik-Haag) [inline pkgs/development/python-modules/django-bootstrap5/default.nix:18]:
```suggestion

  disabled = pythonOlder "3.X";
  
```
you should disable python3 versions older then 3.X what ever the package requires.

**Review by Janik-Haag — CHANGES_REQUESTED** [review]
I just have a small nitpick, this otherwise looks good to me.

File: pkgs/development/python-modules/django-bootstrap5/default.nix (hunk 1 above)
Comment (NetaliDev) [reply pkgs/development/python-modules/django-bootstrap5/default.nix:20]: Fixed

**Review by Janik-Haag — APPROVED** [review]
LGTM; just waiting for ci now.

**Pull request merged**